lib/cloudregister/registerutils.py

```diff
@@ -254,8 +254,12 @@ def fetch_smt_data(cfg, proxies):
         # trying IPv6 first, if available.
         region_servers_ipv4 = []
         region_servers_ipv6 = []
+        region_servers_dns = []
         for srv in region_servers:
-            ip_addr = ipaddress.ip_address(srv.strip())
+            try:
+                ip_addr = ipaddress.ip_address(srv.strip())
+            except ValueError:
+                region_servers_dns.append(srv)
```

Comment:
shouldn't we keep it also with `strip()` (s/srv/srv.strip()) ?

Reply:
Good catch, thanks. Fixed.